[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-1 (#114270)
openshift/openshift-docs: modules/compliance-raw-tailored.adoc

[id="compliance-raw-tailored_{context}"]
= Using raw tailored profiles

[role="_abstract"]

> While the `TailoredProfile` CR enables the most common tailoring operations, the XCCDF standard allows even more flexibility in tailoring OpenSCAP profiles. In addition, if your organization has been using OpenScap previously, you may have an existing XCCDF tailoring file and can reuse it.

Reviewer: ```suggestion
Although the `TailoredProfile` CR enables the most common tailoring operations, you can use the XCCDF standard for more flexibility in tailoring OpenSCAP profiles. 

In addition, if your organization has been using OpenScap previously, you might have an existing XCCDF tailoring file and can reuse it.
```
